src/components/projects/wbs-details/work-package-container/changes-list/changes-list.tsx

```diff
@@ -20,8 +22,17 @@ const ChangesList: React.FC<ChangesListProps> = ({ changes }) => {
       headerRight={<></>}
       list={changes.map((ic) => (
         <>
-          [<Link to={`${routes.CHANGE_REQUESTS}/${ic.changeRequestId}`}>#{ic.changeRequestId}</Link>
-          ] {ic.detail}
+          [<Link to={`${routes.PROJECTS}/${wbsPipe(ic.wbsNum)}`}>{wbsPipe(ic.wbsNum)}</Link>]{' '}
```

Comment:
Why was this changed to link to the wbs element that was changed? I believe this will always route back to itself, thus doing nothing, I think the intention was to link to the change request that resulted in the change to this wbs element, since the changes is a list of changes made to this work package.

Reply:
yeah no clue, I'll switch it back